ast/expressions_test.go

```diff
@@ -131,11 +131,19 @@ func (tc *testExpressionsSuite) createTestCase4ColumnNameExpr() []exprTestCase {
 	}
 }
 
+func (tc *testExpressionsSuite) createTestCase4BetweenExpr() []exprTestCase {
+	return []exprTestCase{
+		{"select b between 1 and 2", "SELECT `b` BETWEEN 1 AND 2"},
```

Comment:
add test case:
`select b between a and b`, `select b between '' and 'b'`,

Reply:
ok
I have added string and time test @leoppro